submissions/kirill8210/popup-task/index.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>html_css_popup</title>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+    <div class="container">
+        <header class="header">
+            <a href="#"><img src="img/favicon.png" alt="favicon"></a>
+            <nav>
+                <ul class="menu">
+                    <li><a href="#">Mail</a></li>
+                    <li><a href="#">Images</a></li>
+                    <li>
+                        <input id="checkbox" class="checkbox" type="checkbox">
+                        <label for="checkbox">
+                            <img src="img/popup-button.png" alt="popup-button">
+                        </label>
+                        <ul class="popup">
+                            <li><a href="#"><img src="img/calendar.png" alt="calendar"><p>Calendar</p></a></li>
```

Comment:
For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)

Reply:
ok

Reply:
Also not fixed

Reply:
ok

Reply:
ok